app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    move_command, from_file, to_file = command.split()
+    with open(from_file) as fr_f:
+        content = fr_f.read()
```

Comment:
U could combine this context manager with the last one, and make you code conciser